transferui/src/test/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModelTest.java

```diff
@@ -192,11 +203,16 @@ public void testCreateTransferViewModel_verifyDefaultValuesWithFundingSource() {
     }
 
     @Test
-    public void testCreateTransferViewModel_verifyDefaultValuesWithoutFundingSource() {
+    public void testInit_withoutFundingSource() {
         CreateTransferViewModel viewModel = new CreateTransferViewModel.CreateTransferViewModelFactory(
                 mTransferRepository, mTransferMethodRepository, mUserRepository
         ).create(CreateTransferViewModel.class);
 
+        CreateTransferViewModel mockedViewModel = spy(viewModel);
```

Comment:
Just create one instance of view model. Do the same in testInit_withFundingSource as well
```suggestion
        CreateTransferViewModel viewModel = spy(new CreateTransferViewModel.CreateTransferViewModelFactory(
                mTransferRepository, mTransferMethodRepository, mUserRepository
        ).create(CreateTransferViewModel.class));

        viewModel.init();
        ... verify and assertions here
```

Reply:
Done